### PR TITLE
annotate TNTRenderMatchModule with ListenerScope

### PR DESCRIPTION
**Pull request by KingOfSquares**
Signed-off-by: KingSimon <19822231+KingOfSquares@users.noreply.github.com>

### CODE REVIEW

File: core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java [hunk 1]
```diff
@@ -17,8 +17,10 @@
 import tc.oc.pgm.api.match.MatchModule;
 import tc.oc.pgm.api.match.MatchScope;
 import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.events.ListenerScope;
 import tc.oc.pgm.util.nms.NMSHacks;
 
+@ListenerScope(value = MatchScope.LOADED)
```
Comment (Electroid) [inline core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java:23]: I think this should actually just be `RUNNING`.

Comment (Electroid) [reply core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java:23]: But if we do this, we need to clean up `entities` when the match is finished. (we can't guarantee the recurring task will execute one last time)

**Review by Electroid — CHANGES_REQUESTED** [review]

File: core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java (hunk 1 above)
Comment (KingOfSquares) [reply core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java:23]: Why should this not run before/after a match? e.g seeing where your TNT _would_ have exploded right after a match ends seems reasonable.

Comment (Electroid) [reply core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java:23]: It's just unnecessary.. a lot of stuff can happen post-cycle, sending a more fake packets to clients for no reason isn't a good idea.

Comment (KingOfSquares) [reply core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java:23]: Ok, ill move it to `Running` and figure out the cleaning thing

Comment (Pugzy) [reply core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java:23]: I agree with KingOfSquares on this one having it scoped to `LOADED`. 
TNT still triggers and chains after the match ends so no reason to stop the module on match end.

Comment (Pablete1234) [reply core/src/main/java/tc/oc/pgm/tntrender/TNTRenderMatchModule.java:23]: I also think it should stay on LOADED, match not running doesn't mean tnt can't still be moving.

**Review by Pablete1234 — APPROVED** [review]

**Review by Electroid — APPROVED** [review]

**Pull request merged**